test/app/users/getManageConsoleServices.test.js

```diff
@@ -0,0 +1,183 @@
+          isExternalService: true,
+          isIdOnlyService: false,
+          isHiddenService: false,
+          isMigrated: false,
+          relyingParty: {},
+        },
+        {
+          id: 'service4Id',
+          name: 'Service 4',
+          description: 'Service for testing purposes',
+          isExternalService: true,
+          isIdOnlyService: false,
+          isHiddenService: false,
+          isMigrated: false,
+          relyingParty: {},
+        },
+        {
+          id: 'service5Id',
+          name: 'Service 5',
+          description: 'Service for testing purposes',
+          isExternalService: true,
+          isIdOnlyService: false,
+          isHiddenService: false,
+          isMigrated: false,
+          relyingParty: {},
+        },
+        {
+          id: 'service6Id',
+          name: 'Service 6',
+          description: 'Service for testing purposes',
+          isExternalService: true,
+          isIdOnlyService: false,
+          isHiddenService: false,
+          isMigrated: false,
+          relyingParty: {},
+        },
+      ]
+    };
+
+    getAllServices.mockReset();
+    getAllServices.mockReturnValue(allServices);
+  });
+
+  it('should call getUserDetails', async () => {
+    await getManageConsoleServices(req, res);
+
+    expect(getUserDetails).toHaveBeenCalled();
+    expect(getUserDetails.mock.calls[0]).toHaveLength(1);
+    expect(sendResult.mock.calls[0][3].user).toMatchObject({
+      id: 'user1',
+    });
+  });
+
+  it('should call getAllServices', async () => {
+    await getManageConsoleServices(req, res);
+
+    expect(getAllServices).toHaveBeenCalled();
+    expect(getAllServices).toReturnWith(
+      { "services": [{ "description": "Service for testing purposes", "id": "service1Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 1", "relyingParty": {} }, { "description": "Service for testing purposes", "id": "service2Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 2", "relyingParty": {} }, { "description": "Service for testing purposes", "id": "service3Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 3", "relyingParty": {} }, { "description": "Service for testing purposes", "id": "service4Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 4", "relyingParty": {} }, { "description": "Service for testing purposes", "id": "service5Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 5", "relyingParty": {} }, { "description": "Service for testing purposes", "id": "service6Id", "isExternalService": true, "isHiddenService": false, "isIdOnlyService": false, "isMigrated": false, "name": "Service 6", "relyingParty": {} }] },

```

Comment:
This is a very long and hard to read line; could the formatting be improved on this one?

Reply:
Agreed, think the formatter must have been playing up on this.